desktop-respons.css

```diff
@@ -0,0 +1,241 @@
+@media screen and (min-width: 768px) {
+  body {
+    background-color: #e5e5e5;
+  }
+
+  .header-at-top {
+    background-image: url(./img/Header\ bg.svg);
```

Comment:
- Hey great job adding the image, however, it's not showing because of the character you've added as this won't be read by the browser or GitHub either, Also, you've downloaded the png and not SVG format, instead, you're used SVG which is not in the downloaded version of your image. when adding images, it's best not to add spaces in their name or extra character. To fix this, kindly rename the image to `Header-bg.png` and then update it in your CSS.